doc/api/process.md

```diff
@@ -3370,7 +3370,7 @@ any exit or close events and without running any cleanup handler.
 
 This function will never return, unless an error occurred.
 
-This function is only available on POSIX platforms (i.e. not Windows or Android).
+This function not available on Windows.
```

Comment:
```suggestion
This function is not available on Windows.
```

Reply:
Since I was not aware of what language to use in the docs for this purpose, I just copied the line used in `process.getuid()`'s documentation for the same. There is same grammatical error over there as well. Should have looked at the grammar as well when copying, silly me!